pytorch/inference/docker/1.13/py3/Dockerfile.cpu

```diff
@@ -159,7 +160,7 @@ RUN pip install --no-cache-dir --extra-index-url https://download.pytorch.org/wh
 # -> PyYaml needs to be high pinned for awscli v1 (current v1 = 1.27.99)
 RUN pip install --no-cache-dir -U -r https://raw.githubusercontent.com/pytorch/serve/v${TORCHSERVE_VERSION}/requirements/common.txt \
     "numpy>=1.22.2,<1.23" \
-    "pyyaml>=5.4,<5.5" \
+    "pyyaml>=6.0" \
```

Comment:
Do the strict version requirements no longer apply for `pyyaml`?

Reply:
it does not as AWSCLI was the limiting factor. After the cpython 3.0 and pyyaml 5.x issue a couple months ago. AWSCLI updated to pyyaml 6.0 with their latest v1.x versions. using the new version does not cause an issue to torchserve. This section is setup to counter the requirements in torchserve as we due need to run higher with out DLC than what the requirements file uses.